app/server.go

```diff
@@ -1201,21 +1201,22 @@ func doCheckNumberOfActiveUsersWarnMetricStatus(a *App) {
 
 	for _, warnMetric := range warnMetrics {
 		data, nErr := a.Srv().Store.System().GetByName(warnMetric.Id)
-		if nErr == nil && data != nil && (data.Value == model.WARN_METRIC_STATUS_ACK || data.Value == model.WARN_METRIC_STATUS_RUNONCE) {
-			mlog.Debug("This metric warning has already been acked or should only run once")
+		if nErr == nil && data != nil && (data.Value == model.WARN_METRIC_STATUS_ACK || (warnMetric.IsBotOnly && data.Value == model.WARN_METRIC_STATUS_RUNONCE)) {
+			mlog.Debug("This metric warning has already been acked or it is bot only and ran once")
 			continue
 		}
 
-		if nErr = a.Srv().Store.System().SaveOrUpdate(&model.System{Name: warnMetric.Id, Value: model.WARN_METRIC_STATUS_LIMIT_REACHED}); nErr != nil {
-			mlog.Error("Unable to write to database.", mlog.String("id", warnMetric.Id), mlog.Err(nErr))
-			continue
-		}
 		warnMetricStatus, _ := a.getWarnMetricStatusAndDisplayTextsForId(warnMetric.Id, nil)
-
 		if !warnMetric.IsBotOnly {
+			//banner and bot metrics - send websocket event
```

Comment:
nit space at start of comment 
```suggestion
			// Banner and bot metrics - send websocket event
```